frontend/src/state/reducers/userPlansReducer.ts

```diff
@@ -288,6 +294,7 @@ export const userPlansReducer = (
       }
       case getType(updateSemesterForActivePlanAction): {
         const { year, season, newSemester } = action.payload;
+        // @ts-ignore
```

Comment:
I think we can do a as any somewhere here instead of the ts-ignore

Reply:
Yeah I was originally trying to make the Plans reducer take IPlanData | ITemplatePlan and the types were pretty much exploding lol ;;-;; I'll remove those